crystal/distribution.yaml

```diff
@@ -396,6 +396,24 @@ repositories:
       url: https://github.com/ros-perception/image_common.git
       version: ros2
     status: maintained
+  joystick_drivers:
+    doc:
+      type: git
+      url: https://github.com/ros2/joystick_drivers.git
+      version: master
```

Comment:
This should be ros2

Reply:
This was definitely me entering them. But please do check the previous ones.